CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# How to contribute #
+
+All help is appreciated. All contributors are unpaid volunteers. If you really want to see something added to Exiv2 - volunteer!
+
+
+## General workflow for contributing ##
+
+1. Fork, then clone the repo:
+```
+git clone git@github.com:your-username/exiv2.git
+```
+
+2. Configure the project and build (More details about how to run CMake at XXX) :
+```
+mkdir build && cd build
+cmake .. && cmake --build . --config Release
+```
+
```

Comment:
That misses the autotools and MSVC build process as alternatives. Could also be an extra file.

Reply:
I can mention that this is the preferred/default/(just the cmake way) way to work in Exiv2 and point to other existing ways that are explained in the README.md. 

My idea is to keep this document as simple as possible, and focus primarily in how to contribute to the project. Details about all the possibilities to configure/build the project can be found in other text files.